scripts/binary/upload-npm-package.coffee

```diff
@@ -20,7 +20,7 @@ uploadFileName = "cypress.tgz"
 isNpmPackageFile = check.extension(npmPackageExtension)
 
 # wonder if our CDN url would just work
-# https://cdn.cypress.io/desktop/0.20.1/osx64/cypress.zip
+# https://cdn.cypress.io/desktop/0.20.1/darwin-x64/cypress.zip
```

Comment:
these two files "upload-npm-package.coffee" and "upload-unique-binary.coffee" have the comments backwards. It should be
- npm package has name `cypress.tgz`
- binary file has name `cypress.zip`